prov/efa/src/rdm/efa_rdm_ep_utils.c

```diff
@@ -619,14 +598,37 @@ ssize_t efa_rdm_ep_trigger_handshake(struct efa_rdm_ep *ep,
  */
 ssize_t efa_rdm_ep_post_handshake(struct efa_rdm_ep *ep, struct efa_rdm_peer *peer)
 {
+	struct efa_rdm_ope *txe;
+	struct fi_msg msg = {0};
 	struct efa_rdm_pke *pkt_entry;
 	fi_addr_t addr;
 	ssize_t ret;
 
 	addr = peer->efa_fiaddr;
+	msg.addr = addr;
+
+	/* ofi_op_write is ignored in handshake path */
+	txe = efa_rdm_ep_alloc_txe(ep, peer, &msg, ofi_op_write, 0,
+				   EFA_RDM_TXE_NO_COMPLETION | EFA_RDM_TXE_NO_COUNTER);
+
+	if (OFI_UNLIKELY(!txe)) {
+		EFA_WARN(FI_LOG_EP_CTRL, "TX entries exhausted.\n");
+		return -FI_EAGAIN;
+	}
+
+	txe->msg_id = -1;
+	/* efa_rdm_ep_alloc_txe() joins ep->base_ep.util_ep.tx_op_flags and passed in flags,
+	 * reset to passed in flags
+	 */
+	txe->fi_flags = EFA_RDM_TXE_NO_COMPLETION | EFA_RDM_TXE_NO_COUNTER;
```

Comment:
Probably not needed - I think we can remove them.

Reply:
Done in both places... my only concern is that the user can potentially set flags that make us not behave as expected... (like FI_DELEIVERY_COMPLETE)... I think our tests should catch this though?

Reply:
I tested my theory in the debugger:
```
#define FI_DELIVERY_COMPLETE	(1ULL << 28)

p  tx_op_flags & (1ULL << 28)
$8 = 268435456

#define FI_COMPLETION		(1ULL << 24)

p  tx_op_flags & (1ULL << 24)
$9 = 16777216
```

Looks like we do need these so the common utilities work correctly.